cmd/jsonApplyPatch.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/evanphx/json-patch"
+)
+
+func jsonApplyPatch(config jsonApplyPatchOptions, telemetryData *telemetry.CustomData) {
+	err := runJsonApplyPatch(&config, &piperutils.Files{})
+	if err != nil {
+		log.Entry().WithError(err).Fatal("step execution failed")
+	}
+}
+
+func runJsonApplyPatch(config *jsonApplyPatchOptions, fileUtils piperutils.FileUtils) error {
+	schemaFile, err := fileUtils.FileRead(config.Input)
+	if err != nil {
+		return nil
+	}
+	schema := []byte(string(schemaFile))
```

Comment:
The variable `schemaFile` is already of type `[]byte`.
```suggestion
	schema, err := fileUtils.FileRead(config.Input)
	if err != nil {
		return nil
	}
```